hydra-node/test/Hydra/Chain/Direct/Contract/FanOut.hs

```diff
@@ -74,13 +75,20 @@ healthyContestationDeadline :: UTCTime
 healthyContestationDeadline =
   slotNoToUTCTime $ healthySlotNo - 1
 
+healthyContestationPeriod :: OnChain.ContestationPeriod
+healthyContestationPeriod = OnChain.contestationPeriodFromDiffTime $ fromInteger healthyContestationPeriodSeconds
+
+healthyContestationPeriodSeconds :: Integer
+healthyContestationPeriodSeconds = 10
```

Comment:
Could: not put this on module level. These symbols are only used in `healthyFanoutDatum` so it's fine to keep them there in a `where` block.